luaclang.c

```diff
@@ -204,13 +345,56 @@ static int cursor_visitchildren(lua_State *L)
         return 0;
 }
 
-void new_metatable(lua_State *L, const char *name, luaL_Reg *reg)
+/* -- Type functions -- */
+
+/*
+        Format - cur_type:getTypeSpelling()
```

Comment:
Ins't it just `type:getSpelling`?

Reply:
@gligneul There are different clang functions for `type` as well as `cursor` objects i.e `clang_getCursorSpelling(cursor)` and `clang_getTypeSpeliing(type)`

Reply:
Yes, but the method names could be the same. There won't be any conflicts because they will be placed in different metatables.

Reply:
Okay. Thought of naming them resembling the clang function names. Will change